.travis.yml

```diff
@@ -1,9 +1,12 @@
 # Set Ruby as the language so it doesn't download the pip things. Instead, let docker do that.
 language: ruby
 cache: bundler
+install:
+  - docker-compose -f travis-docker-compose.yml build
```

Comment:
I think if you ask docker to run a container that is hasn't yet built it will build it first automatically, so that:

```
docker-compose run web tox
```

is more or less equivalent to

```
docker-compose build && docker-compose run web tox
```

basically what adding the `build` command does is separates building the `watch` and `web` containers from running the test commands, so we don't have to look at all the output from installing packages and pulling the containers from docker hub.